src/schematic/schematic_component.ts

```diff
@@ -127,6 +128,7 @@ export const schematic_component = z.object({
   symbol_display_value: z.string().optional(),
   subcircuit_id: z.string().optional(),
   schematic_group_id: z.string().optional(),
+  is_schematic_group: z.boolean().optional(),
```

Comment:
you need to add `source_group_id: z.string().optional()` and make `source_component_id` optional